web/signer.ts

```diff
@@ -0,0 +1,137 @@
+import * as jsrsasign from 'jsrsasign';
```

Comment:
Should we have some test code for this?

Reply:
@feeblefakie 
Thank you for the reminder!
Yes. We will have a test for that in another PR.
I didn't put it in this PR is because to test WebCrypto we need to launch a headless browser. It needs some additional setup. I tried to reduce the PR size.